src/components/page-components/ExplorePage/ExplorePage.css

```diff
@@ -3,6 +3,11 @@
   animation-name: slideDown;
 }
 
+.explore-page--card-container {
+  display: grid;
```

Comment:
I'd say using the global class here .grid as it contains padding extra that will make it consistent with other grids